### PR TITLE
 Adding the Sparkline chart unit testcase into the chartsV9.

### PR DESCRIPTION
**Pull request by v-asurse**
## Previous Behavior

Sparkline chart unit testcase is not present in there.

## New Behavior

 Added the Sparkline chart unit testcase with some configuration changes.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1987,6 +1987,36 @@
     "@griffel/react" "^1.5.14"
     "@swc/helpers" "^0.5.1"
 
+"@fluentui/react-shared-contexts@^9.19.0":
```
Comment (Hotell) [inline yarn.lock:1990]: this looks like leftover from bad merge or something ? please undo these changes

**Review by Hotell — CHANGES_REQUESTED** [review]
please address my comments in order to proceed. ty

File: yarn.lock (hunk 1 above)
Comment (v-asurse) [reply yarn.lock]: I have undone those changes. ty.

File: scripts/constants.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
```
[the remark this answers is not in the document]
Comment (v-asurse) [reply scripts/constants.js]: Yes, I remove that file from "scripts/folder" location.

**Review by AtishayMsft — APPROVED** [review]
